inc/compatibility/woocommerce.php

```diff
@@ -146,6 +148,48 @@ public function add_customizer_options( $options ) {
 		return $options;
 	}
 
+	/**
+	 * Add inline style for payment methods.
+	 */
+	public function payment_style() {
+		if ( ! class_exists( 'WooCommerce', false ) ) {
+			return;
+		}
+
+		if ( ! class_exists( 'WC_Payment_Gateways', false ) ) {
+			return;
+		}
+
+		$wc_gateways      = new \WC_Payment_Gateways();
+		$payment_gateways = $wc_gateways->get_available_payment_gateways();
+		if ( empty( $payment_gateways ) ) {
+			return;
+		}
+
+		$css = '';
+
+		// Inline style for stripe.
+		if ( array_key_exists( 'stripe', $payment_gateways ) && property_exists( $payment_gateways['stripe'], 'settings' ) && array_key_exists( 'payment_request_button_locations', $payment_gateways['stripe']->settings ) ) {
```

Comment:
We can do early returns here. I think using `isset` checks would be better, as it will account for the value of the key/property not being null.

```suggestion
if( ! isset( $payment_gateways['stripe'] ) {
  return;
}

if( ! isset( $payment_gateways['stripe']->settings ) {
  return;
} 
```